lib/assetcompute.js

```diff
@@ -161,8 +172,13 @@ class AssetCompute {
             retryOptions: this.retryOptions
         });
         if (!response.ok) {
-            const msg = await response.text();
-            throw Error(`Unable to invoke /process: ${response.status} ${msg}`);
+            const responseText = await response.text();
```

Comment:
You could make those few lines of code a function with the invoked endpoint as parameter to avoid the code repetitions.

Reply:
Done